### DISCUSSION TITLE
How to add a superuser privilege to a postgres user

### DISCUSSION THREAD

**Discussion by spyriaev** [supabase/supabase#12535] [Questions] [answered]
I can't add a superuser privilege to a new db through this command:

`alter user postgres with superuser`

I get this error:

`Failed to run sql query: must be superuser to alter superuser roles or change superuser attribute`

I ran the command in Supabase SQL Editor.

I ran the command successfully in a db that I created a month ago with the result:

`Success. No rows returned`

Let me know where I am going wrong.

**Comment by GaryAustin1** [comment, marked as answer]
Supabase is modifying all instances to no longer have superuser privileges...
https://github.com/supabase/supabase/discussions/9314

**Reply by spyriaev** [reply]
Got it, thx!
